src/frontend/App/context/BehandlingContext.tsx

```diff
@@ -58,6 +62,9 @@ const [BehandlingProvider, useBehandling] = constate(() => {
         hentTotrinnskontroll,
         hentBehandlingshistorikk,
         regler,
+        antallIRedigeringsmodus,
+        settAntallIRedigeringsmodus,
+        ulagretData,
```

Comment:
Virker litt unødvendig å eksponere `antallIRedigeringsmodus`. 
Alternativt kan vi eksponere:
- `økAntallIRedigeringsmodus`, `reduserAntallIRedigeringsmodus` og `nullstillAntallIRedigeringsmodus` ?

 Kanksje litt overkill, men kunne vært en hook (`useRedigeringsmodus`) som returnerte disse samt `harUlagretData` ? 

Andre som har tanker?

Reply:
Ahh, sånn at man kaller hver av de ulike funksjonene for å endre på `antallIRedigeringsModus` ? 
Det blir bittelitt mer kode, men da vil koden kanskje være litt mer selvforklarende enn å bruke tall direkte! Ikke dårlig innspill! :)

Reply:
Ja, det blir ikke mindre kode, men tar bort behovet for å vite noe om antall når man skal endre verdien